runtime/compiler/arm/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -740,7 +756,9 @@ uint32_t J9::ARM::AheadOfTimeCompile::_relocationTargetTypeToHeaderSizeMap[TR_Nu
    0,                                        // TR_NativeMethodAbsolute                = 56,
    0,                                        // TR_NativeMethodRelative                = 57,
    16,                                       // TR_ArbitraryClassAddress               = 58,
-   28                                        // TR_DebugCounter                        = 59
+   28,                                       // TR_DebugCounter                        = 59
+   0,                                        // TR_ClassUnloadAssumption               = 60
```

Comment:
is TR_ClassUnloadAssumption supposed to be in this pull request?

Reply:
I see, you're also co-dependent on #2672 (one of you will conflict with whichever is merged first :) ).

Reply:
`TR_ClassUnloadAssumption` is an existing `TR_ExternalRelocationTargetKind` with value 60. It wasn't in the size tables previously because it's not for AOT:

    TR_ClassUnloadAssumption               = 60, // this should not be used in AOT relocations

I need to specify a value there in order to get to index 61 (`TR_J2IVirtualThunkPointer`).